python/pynq/mmio.py

```diff
@@ -38,6 +38,7 @@
 import mmap
 import math
 from . import general_const
+import numpy as np
```

Comment:
The use of numpy is kind of an overkill, since we do not do arithmetic over the data. I am not sure if we really need this.